src/List.js

```diff
@@ -13,7 +13,28 @@ import { Link } from 'react-router-dom';
 import { NavigationMenu } from './NavigationMenu';
 import { useHistory } from 'react-router-dom';
 
-const convertToDays = (num) => num / 1000 / 60 / 60 / 24;
+const convertToDays = (num) => Math.round(num / 1000 / 60 / 60 / 24);
+
+const daysSinceLastPurchaseOrCreationTime = (item) =>
+  convertToDays(new Date() - (item.lastPurchasedDate || item.creationTime));
+
+const itemIsInactive = (item) =>
+  daysSinceLastPurchaseOrCreationTime(item) > 2 * item.previousEstimate ||
+  item.totalPurchases === 1;
```

Comment:
Good catch, thank you!
@gabynr so I guess we'd just change to 

item.totalPurchases === 1 ||  item.totalPurchases === 0

or 
item.totalPurchases <= 1 ??

Do you have a preference?

Reply:
we can't set it in `zero`, because in the inactive criteria  says the following
 `Inactive (when there’s only 1 purchase in the database or the purchase is really out of date [the time that has elapsed since the last purchase is 2x what was estimated])`  
and also we can't set in `zero` because all the new items would be inactive.

Reply:
yes but shouldn't new items be inactive, since they aren't bought yet?

Reply:
No. when the item is added, the user choose the following: `soon (7 days)`,` kind of soon ` (14 days), and `not soon` (30 days), which means you have an estimate per purchase. but if you mark this element for the first time, the application in 24 hours unchecks the element because it is the time we create to uncheck that element, and also the app supposes that the user bought that element and now it is inactive because the user doesn't need that item very soon. Can you see now?